browser/src/control/jsdialog/Definitions.Types.ts

```diff
@@ -39,6 +39,7 @@ interface JSBuilderOptions {
 	// modifiers
 	noLabelsForUnoButtons: boolean; // create only icon without label
 	useInLineLabelsForUnoButtons: boolean; // create labels next to the icon
+	suffix: any; // add a suffix to the element ID to make it unique among different builder instances.
```

Comment:
type: string , please do not use any where possible

Reply:
Oh, it's a typed language; I always forget.